src/script/sign.cpp

```diff
@@ -620,7 +620,12 @@ bool PublicOnlySigningProvider::GetCScript(const CScriptID &scriptid, CScript& s
     return m_provider->GetCScript(scriptid, script);
 }
 
-bool PublicOnlySigningProvider::GetPubKey(const CKeyID &address, CPubKey& pubkey) const
+bool PublicOnlySigningProvider::GetPubKey(const CKeyID& keyid, CPubKey& pubkey) const
 {
     return m_provider->GetPubKey(address, pubkey);
```

Comment:
In commit "Make SigningProvider expose key origin information"

You should change `address` to `keyid` in this commit instead of "Generalize PublicOnlySigningProvider into HidingSigningProvider"

Reply:
Done.